exo-go/src/cmd/configure.go

```diff
@@ -110,8 +111,106 @@ var configureCreateCmd = &cobra.Command{
 	},
 }
 
+var configureUpdateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Updates secret key entries in remote secrets store",
+	Long:  "Updates secret key entries in remote secret store. Keys should already exist.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Print("We are about update keys in the remote store!\n\n")
+
+		secretsBucket, awsRegion, err := getSecretsConfig()
+		if err != nil {
+			log.Fatalf("Cannot update secrets: %s", err)
+		}
+
+		secretsString, err := aws.ReadSecrets(secretsBucket, awsRegion)
+		if err != nil {
+			log.Fatalf("Cannot read secrets: %s", err)
+		}
+		existingSecrets := types.NewSecrets(secretsString)
+
+		newSecrets := map[string]string{}
+		for true {
+			secretName := prompt.String("Secret name (leave blank to finish prompting)")
+			if secretName == "" {
+				break
+			}
+			if _, hasKey := existingSecrets[secretName]; !hasKey {
+				fmt.Printf("Secret for '%s' does not exists. Use 'exo configure create' to create it.\n\n", secretName)
+			} else {
+				secretValue := prompt.StringRequired("Secret value")
+				newSecrets[secretName] = secretValue
+			}
+		}
+
+		if len(newSecrets) > 0 {
+			secretsPretty, err := json.MarshalIndent(newSecrets, "", "  ")
+			if err != nil {
+				log.Fatalf("Could not marshal secrets map: %s", err)
+			}
+			fmt.Print("You are updating these secrets:\n\n")
+			fmt.Printf("%s\n\n", string(secretsPretty))
+
+			if ok := prompt.Confirm("Do you want to continue?"); ok {
+				err = aws.MergeAndWriteSecrets(existingSecrets, newSecrets, secretsBucket, awsRegion)
+				if err != nil {
+					log.Fatalf("Cannot update secrets: %s", err)
+				}
+			} else {
+				fmt.Println("Secret update abandoned.")
+			}
+		}
+	},
+}
+
+var configureDeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Deletes secrets from the remote secrets store",
+	Long:  "Deletes secrets from the remote secrets store. Ignores any keys passed in that don't exist on the remote store.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Print("We are about to delete secrets from the secret store...\n\n")
+
+		secretsBucket, awsRegion, err := getSecretsConfig()
+		if err != nil {
+			log.Fatalf("Cannot delete secrets: %s", err)
+		}
+
+		secretKeys := []string{}
+		for true {
+			secretName := prompt.String("Secret name (leave blank to finish prompting)")
+			if secretName == "" {
+				break
+			} else {
+				secretKeys = append(secretKeys, secretName)
+			}
```

Comment:
I don't have a strong feeling either way. There shouldn't be too many secrets, only things like db credentials and api keys. The AWS client already ignores any secrets you try to delete that don't exist. Even if we write our own validator that seems like a lot of extra overhead just to essentially ignore the secret as well right (which AWS is already doing)?